pkg/odo/cli/component/create.go

```diff
@@ -7,6 +7,8 @@ import (
 	"path/filepath"
 	"strings"
 
+	segment "github.com/openshift/odo/pkg/segment/context"
```

Comment:
This has probably been set by your IDE, but is confusing. You should instead write `context.SetComponentType` in line 911